blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/preload/FavoritesStatePreloader.java

```diff
@@ -54,9 +60,10 @@
     @Override
     protected FetchData getFetchData(@Nonnull BlueUrlTokenizer blueUrl) {
         User jenkinsUser = User.current();
+        BlueOrganization organization = Iterables.getFirst(OrganizationFactory.getInstance().list(), null);
```

Comment:
There must always be at least one. Getting the first organization is really a temporary thing until we have a strategy for the preloaders. @vivek  does not like by now using the url to get the organization we are at.